src/message/from_prior/pack.rs

```diff
@@ -114,6 +116,41 @@ impl FromPrior {
 
         Ok((from_prior_jwt, String::from(kid)))
     }
+
+    pub(crate) fn validate_pack(&self, issuer_kid: Option<&str>) -> Result<()> {
```

Comment:
Do we need it public?

Reply:
Yes, because it is also used in `Message._validate_pack_plaintext`.